rtgui/cropwindow.cc

```diff
@@ -1623,7 +1623,7 @@ void CropWindow::expose (Cairo::RefPtr<Cairo::Context> cr)
                     const int shThreshold = options.shadowThreshold;
                     const float ShawdowFac = 64.f / (options.shadowThreshold + 1);
                     const float HighlightFac = 64.f / (256 - options.highlightThreshold);
-                    const bool showclippedAny = (!showR && !showG && !showB && !showL); // will show clipping if any of RGB chanels is clipped
+                    const bool showclippedAny = (!showR && !showG && !showB && !showL); // will show clipping if any (all) of RGB chanels is (shadow) clipped
```

Comment:
It shouldn't be called `showclippedAny` anymore, but `showClippedAll`, and "any (all)" in the comment doesn't really make sense, more like
`// will show clipping if all RGB channels are below the shadow threshold`

Reply:
showclippedAny is also used for highlight clipping, where it still does what was in the original comment

Reply:
So `showClipped` with the comment `if all RGB channels are below the shadow threshold or either channel is above the highlights threshold`

Reply:
`showClippedNormal` or what the bool value really reflects: `displayAllColors`